code/game/objects/structures/manned_turret.dm

```diff
@@ -47,10 +47,10 @@
 	if(user.incapacitated() || !istype(user))
 		return
 	if(do_after(user, 30, target = M))
+		M.forceMove(get_turf(src))
 		. = ..()
 		if(!.)
 			return
-		M.forceMove(get_turf(src))

```

Comment:
Them not being on the proper turf was causing . = FALSE and so nobody could mount the gun